maintainers/depmgmt.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: default
+nav_order: 4
+parent: RAPIDS Maintainer Docs
+title: Dependency Management
+---
+
+# Dependency Management Documentation
+
+## Overview
+
+Outlines how to test and change dependencies of the RAPIDS projects
+
+### Intended audience
+
+Developers
+{: .label .label-green}
+
+Project Leads
+{: .label .label-blue}
+
+Operations
+{: .label .label-purple}
+
+## Testing a Dependency
+
+The build scripts located in the `ci/gpu` folder of the project can be easily modified to test a dependency. For example, the build script for cuDF may contain these lines to install packages required to build from source and test notebooks:
```

Comment:
Add a line thats like: If you want to test a a new dependency before adding/updating the integration repo, follow these steps